src/targets/gpu/include/migraphx/gpu/context.hpp

```diff
@@ -323,12 +323,12 @@ struct context
         return std::make_pair(nullptr, nullptr);
     }
 
-    float get_elapsed_ms() const
+    static float get_elapsed_ms(hipEvent_t start, hipEvent_t stop)
```

Comment:
Don't we want to keep non-static method if we need in future ?

Reply:
I dont think we can overload a static and non-static function. If we do need to the method in the future then we can think about hot to name this better. Most likely we wont need it in the future as launching with the events attached has too much overhead and probably wont be fixed soon.